.github/workflows/container-image.yml

```
@@ -36,10 +64,48 @@ jobs:
       - name: Check out the repo
         uses: actions/checkout@v3
 
-      - name: Build and push
+      - name: Build and push operator
+        id: build-push-operator
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          push: true
+          tags: ${{ steps.meta-operator.outputs.tags }}
+          labels: ${{ steps.meta-operator.outputs.labels }}
+
+      - name: Setup Go ${{ env.GOLANG_VERSION }}
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ env.GOLANG_VERSION }}
```

Not sure we need an environment variable here.
```suggestion
      - name: Setup Go
        uses: actions/setup-go@v3
        with:
          go-version: 1.18
```

---

You won't have Go 1.18 in RHEL before RHEL 8.7, i.e. Nov 2022.

---

Just to have the same version in other steps if we need it. I agree it's a stretch.